### PR TITLE
Add LEDAcrypt

### PR DESCRIPTION
**Pull request by TomerHawk**
updated to latest status

### CODE REVIEW

File: encryption/ledacrypt/ledacrypt-pkc-cca/bench/script [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (TomerHawk) [reply encryption/ledacrypt/ledacrypt-pkc-cca/bench/script:1]: :D

File: encryption/ledacrypt/ledacrypt-pkc-cca/ledacrypt-pkc-cca.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: LEDAcrypt PKC
+type: KTM
+security notion: IND-CCA
+sources:
+  - 'https://www.ledacrypt.org/documents/LEDAcrypt_v3.pdf, page 80,81,113,114'
```
[the remark this answers is not in the document]
Comment (TomerHawk) [reply encryption/ledacrypt/ledacrypt-pkc-cca/ledacrypt-pkc-cca.yaml:5]: Maybe in the description field or do you mean it should be somehow highlighted?

[the remark this answers is not in the document]
Comment (taudor) [reply encryption/ledacrypt/ledacrypt-pkc-cca/ledacrypt-pkc-cca.yaml:5]: Following our other discussions, I would say that we should not include the PKC scheme. Despite having parameter sets and benchmarks this time.

File: encryption/ledacrypt/ledacrypt-pkc-cca/impl/avx2.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+name: AVX2
+platform: Intel Core i5-6600 3.2  GHz
```
Comment (taudor) [inline encryption/ledacrypt/ledacrypt-pkc-cca/impl/avx2.yaml:2]: `impl/platform` is the programming platform, so here something like `C11`(?).
And `bench/platform` is the actual hardware.

File: encryption/ledacrypt/ledacrypt-pkc-cca/bench/avx2_ledapkc3364_opt.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+sources:
+  - 'https://www.ledacrypt.org/documents/LEDAcrypt_v3.pdf#page=120'
+platform: C11
```
Comment (taudor) [inline encryption/ledacrypt/ledacrypt-pkc-cca/bench/avx2_ledapkc3364_opt.yaml:3]: hardware information missing

File: encryption/ledacrypt/ledacrypt-pkc-cca/bench/avx2_ledapkc32sl_opt.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+sources:
+  - 'https://www.ledacrypt.org/documents/LEDAcrypt_v3.pdf#page=120'
+platform: C11
```
Comment (taudor) [inline encryption/ledacrypt/ledacrypt-pkc-cca/bench/avx2_ledapkc32sl_opt.yaml:3]: hardware information missing

File: encryption/ledacrypt/ledacrypt-kem-cca/impl/avx2.yaml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+name: AVX2
+platform: Intel Core i5-6600 3.2  GHz
```
Comment (taudor) [inline encryption/ledacrypt/ledacrypt-kem-cca/impl/avx2.yaml:2]: Should be something like `c`.

File: encryption/ledacrypt/ledacrypt-kem-cca/bench/avx2_ledakem1464_opt.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+platform: C11
```
Comment (taudor) [inline encryption/ledacrypt/ledacrypt-kem-cca/bench/avx2_ledakem1464_opt.yaml:1]: hardware information missing

File: encryption/ledacrypt/ledacrypt-pkc-cca/impl/avx2.yaml (hunk 3 above)
Comment (TomerHawk) [reply encryption/ledacrypt/ledacrypt-pkc-cca/impl/avx2.yaml:2]: Oh right, this has to be one of the early schemes I worked on.

File: encryption/ledacrypt/ledacrypt-pkc-cca/ledacrypt-pkc-cca.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomerHawk) [reply encryption/ledacrypt/ledacrypt-pkc-cca/ledacrypt-pkc-cca.yaml:5]: So you want to include the parameter sets and benchmarks of PKC but not the scheme itself, doesn't it cause irritations if they will be part of one of the KEM flavors?

**Review by taudor — APPROVED** [review]

**Pull request merged**